src/pages/partner/index.md

```diff
@@ -41,173 +44,149 @@ The following workflow diagram describes the steps in the partner registration p
 
-    ![Add Events](resources/ss_Add_Events.png)
+   ![Add Events](resources/ss_Add_Events.png)
 
 5. Select the **Globalization Content Service Event** and click **Next**.
 
-    ![Add Globalization Content Service Events](resources/ss_Add_Events_GCS.png)
-
-6. The **Configure event registration** screen shows four events that you can subscribe.      
-   1. **LEVERAGE\_TM**: Used to extract text for localization out of an Adobe asset. Also, access the translation memory (TM) to enable the optimum re-use of a previously translated content. 
-   2. **PROJECT\_UPDATE**: Used to update the project configuration information. The configuration data can include info on locales, content provider system and so on. 
-   3. **TRANSATE**: Used to trigger an event when a task is sent for localization.
-   4. **UPDATE\_TM**: Used to update the translation memory. During the asset completion of manual translation, translated strings are saved in TM so that they can be used later if similar strings appear in a future task. It helps to reduce the cost of manual translation.
-   
-   ![Configure events](resources/ss_Configure_event_registration.png)   
-
-7. Select the events you want to subscribe and press **Next**.
-
-    ![Select event](resources/ss_Configure_event_registration_2.png)      
+   ![Add Globalization Content Service Events](resources/ss_Add_Events_GCS.png)
 
-    Create a new service account (JWT) credential. Here, you create a JSON Web Token (JWT) that includes your credentials. You’ll begin your sessions by exchanging the JWT for an access token at Adobe I/O console. Once access is granted, you start using Globalization Content Service events. Since you created a public key while registering to Globalization Content Service APIs, you can use it to create JWT.
+6. The **Configure event registration** screen shows four events that you can subscribe.
 
-8. Select **Upload your public key.**
-
-    ![Create JWT event](resources/ss_Create_JWT_event.png)
+   1. **LEVERAGE_TM**: Used to extract text for localization out of an Adobe asset. Also, access the translation memory (TM) to enable the optimum re-use of a previously translated content.
+   2. **PROJECT_UPDATE**: Used to update the project configuration information. The configuration data can include info on locales, content provider system and so on.
+   3. **TRANSATE**: Used to trigger an event when a task is sent for localization.
+   4. **UPDATE_TM**: Used to update the translation memory. During the asset completion of manual translation, translated strings are saved in TM so that they can be used later if similar strings appear in a future task. It helps to reduce the cost of manual translation.
 
-9. Drag and drop the public key (***certificate\_pub.crt***) and select **Next**.
+   ![Configure events](resources/ss_Configure_event_registration.png)
 
-    ![Upload Public Key](resources/ss_Upload_Public_Key.png)
+7. Select the events you want to subscribe and press **Next**.
```

Comment:
Select the events you want to subscribe *to*

Reply:
done